content/news/029/index.md

```diff
@@ -190,6 +190,19 @@ Version 0.9.0 brings a few breaking changes. Notably `wgpu` was updated to
 [pixels]: https://github.com/parasyte/pixels
 [pixels-changelog]: https://github.com/parasyte/pixels/releases/tag/0.9.0
 
+### [bevy-remote-devtools]
+
+![Demo of the tauri based development app](bevy-remote-devtools.gif)
+
+[bevy-remote-devtools] is a plugin and UI application for the [Bevy] game engine allowing to view entities and their components, asset resources like meshes, events from the [tracing] crate and system timings using a very basic profiler. It also supports all of that over network so debugging can be done from any remote machine and vice versa.
```

Comment:
I'm not sure if `[Bevy]` will link to `[bevy]` (and I can't test myself right now). Can someone check this out?

Reply:
It did for me but I changed cases anyway to be safe. It was just the result of my auto formatting it seems anyway.

Reply:
I think `[Bevy]` worked since there is another project already using that link. Hopefully Zola 0.15 will help catch these problems in CI :)